pylint/utils/utils.py

```diff
@@ -0,0 +1,376 @@
+# -*- coding: utf-8 -*-
```

Comment:
Feels odd to have an `utils.py` inside an `utils` package, but I don't have a better suggestion.

Reply:
This permit to change the `from pylint import utils` to `from pylint.utils import utils, AstWalker, ...` and the class can be imported one by one while still using the `utils.function_name()` for functions. It makes some change elsewhere less painful. Ideally every utils function would be in a semantically significant module or package but there is a lot of those utility functions...